aws/eks-cluster/locals.tf

```diff
@@ -3,4 +3,6 @@ locals {
 
   cluster_name = "${var.name}-eks-${var.environment}"
   namespaces   = { for namespace in var.custom_namespaces : namespace => namespace }
+
+  datadog_agent_cluster_role_name = "datadog-agent"
```

Comment:
Please use variable

Reply:
should it be a required variable or a variable with a default value?
We don't need to output this variable, if it doesn't have a default value.

Reply:
updated